src/docfx/lib/StringUtility.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class StringUtility
```

Comment:
consider calling this `Levenshtein.GetLevenshteinDistance` and does not make it an extension method